src/v/redpanda/admin/util.cc

```diff
@@ -36,4 +57,24 @@ bool get_boolean_query_param(
            || str_param == "1";
 }
 
+bool path_decode(const std::string_view& in, ss::sstring& out) {
```

Comment:
nitpick: Prefer to pass `std::string_view` by value.

Reply:
https://github.com/redpanda-data/redpanda/pull/16694#discussion_r1504531137